cumulusci/core/config/BaseProjectConfig.py

```diff
@@ -177,42 +172,44 @@ def repo_info(self):
             url_info = self._split_repo_url(repo_url)
             info.update(url_info)
 
-        # If running in a CI environment, make sure we have all the needed
-        # git info or throw a ConfigError
-        if info["ci"]:
-            validate = OrderedDict(
-                (
-                    # <key>, <env var to manually override>
-                    ("branch", "CUMULUSCI_REPO_BRANCH"),
-                    ("commit", "CUMULUSCI_REPO_COMMIT"),
-                    ("name", "CUMULUSCI_REPO_URL"),
-                    ("owner", "CUMULUSCI_REPO_URL"),
-                    ("root", "CUMULUSCI_REPO_ROOT"),
-                    ("url", "CUMULUSCI_REPO_URL"),
-                )
+    def _override_repo_env_var(self, repo_env_var, local_var, info):
```

Comment:
@davisagli - You know of a way that this method could be modified so that it handles CUMULUSCI_REPO_URL too? Right now it has a couple of 'extra steps' that the others don't.